File: pset1/mario (less confortable).c
// Draws a pyramid using recursion

#include <cs50.h>
#include <stdio.h>

void draw(int n);

int main (void)
{
    // Get height of pyramid
    int height = get_int("Height: ");

    // Draw pyramid
    draw(height);
}

void draw(int n)
{
    // Draw one more row of width n
    for(int i = 0; i < n; i++)
    {
        for (int j = n - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario (less confortable).py
# Draw right aligned pyramid as in Mario

from cs50 import get_int

# Continuously control input: 0 < int < 9
while True:
    height = get_int("Height: ")
    if 0 < height < 9:
        break
    else:
        print("Height must be between 0 and 9 excluded")

# Draw one more row of width n
for i in range(height):
    print(" " * (height - i - 1) + "#" * (i + 1))
